.gitignore

```diff
@@ -13,7 +13,8 @@ Cargo.lock
 # MSVC Windows builds of rustc generate these, which store debugging information
 *.pdb
 
-cmd/ef_tests/vectors
+cmd/ef_tests/ethereum_rust/vectors
+cmd/ef_tests/levm/vectors
```

Comment:
Question: Aren't the test vectors the same? Can't we use the same files for both tests setups?

Reply:
Not quite, we use only a subset of the EF tests for the VM, the `GeneralStateTests` specifically.

Reply:
Nevertheless, I think that we can do a proper review of this. I'll file an issue to tackle this.

Reply:
Filed in https://github.com/lambdaclass/lambda_ethereum_rust/issues/1173.

Reply:
Great! 🚀